components/ErrorDisplay/__tests__/__snapshots__/ErrorDisplay.test.js.snap

```diff
@@ -2,22 +2,22 @@
 
 exports[`ErrorDisplay should render with just required props 1`] = `
 <div
-  className="ErrorDisplay"
+  className="ErrorDisplay bg-[#121212] w-[100% h-[100%]"
```

Comment:
Can you add our variables.css to the theme object? You can prefix them with `theme` so that intellisense is helpful. So, here you would end up with `bg-themeGray` instead of `bg-[#121212]`

Also:
```suggestion
  className="ErrorDisplay bg-[#121212] w-full h-full"
```